ios/engine/KMEI/KeymanEngine/en.lproj/Localizable.stringsdict

```diff
@@ -0,0 +1,128 @@
+<dict>
+	<key>menu-langsettings-lexical-model-count</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@count@</string>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>zero</key>
+			<string>No dictionaries installed</string>
+			<key>one</key>
+			<string>%u dictionary installed</string>
+			<key>other</key>
+			<string>%u dictionaries installed</string>
+		</dict>
+	</dict>
+	<key>notification-update-failed</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@count@</string>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>one</key>
+			<string>%u update failed</string>
+			<key>other</key>
+			<string>%u updates failed</string>
+		</dict>
+	</dict>
+	<key>notification-update-success</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@count@</string>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>one</key>
+			<string>%u update successful</string>
+			<key>other</key>
+			<string>%u updates successful</string>
+		</dict>
+	</dict>
+	<key>settings-keyboards-installed-count</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@count@</string>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>zero</key>
+			<string>No keyboards installed</string>
+			<key>one</key>
+			<string>%u keyboard installed</string>
+			<key>other</key>
+			<string>%u keyboards installed</string>
+		</dict>
+	</dict>
+	<key>settings-languages-installed-count</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@count@</string>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>zero</key>
+			<string>No languages installed</string>
+			<key>one</key>
+			<string>%u language installed</string>
+			<key>other</key>
+			<string>%u languages installed</string>
+		</dict>
+	</dict>
+	<key>package-default-found-keyboards</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@count@</string>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>u</string>
+			<key>zero</key>
+			<string>Found no keyboards in package</string>
+			<key>one</key>
+			<string>Found %u keyboard in package:</string>
+			<key>other</key>
+			<string>Found %u keyboards in package:</string>
```

Comment:
Are the `:`s a separator for another string that follows? I notice only these package strings have the colons

Reply:
https://github.com/keymanapp/keyman/blob/836fd0b92187bcf4e5e9a739332e4bb7c95eb4b1/ios/engine/KMEI/KeymanEngine/Classes/KeyboardKeymanPackage.swift#L33-L40